src/NzbDrone.Core/ImportLists/AniList/List/AniListSettings.cs

```diff
@@ -0,0 +1,72 @@
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+
+namespace NzbDrone.Core.ImportLists.AniList.List
+{
+    public class AniListSettingsValidator : AniListSettingsBaseValidator<AniListSettings>
+    {
+        public AniListSettingsValidator()
+        : base()
+        {
+            RuleFor(c => c.Username).NotEmpty();
+
+            RuleFor(c => c.ImportCurrent).NotEmpty()
+                                         .WithMessage("At least one status type must be selected")
+                                         .When(c => !(c.ImportPlanning || c.ImportCompleted || c.ImportDropped || c.ImportPaused || c.ImportRepeating));
+        }
+    }
+
+    public class AniListSettings : AniListSettingsBase<AniListSettings>
+    {
+        public const string sectionImport = "Import List Status";
+        public const string unitListStatus = "List Status";
+        public const string unitMediaStatus = "Media Status";
+
+        public AniListSettings()
+        : base()
+        {
+            ImportCurrent = true;
+            ImportPlanning = true;
+            ImportReleasing = true;
+            ImportFinished = true;
+        }
+
+        protected override AbstractValidator<AniListSettings> Validator => new AniListSettingsValidator();
+
+        [FieldDefinition(1, Label = "Username", HelpText = "Username for the List to import from")]
+        public string Username { get; set; }
+
+        [FieldDefinition(0, Label = "Import Watching", Type = FieldType.Checkbox, Section = sectionImport, Unit = unitListStatus)]
```

Comment:
I don't think using unit for this makes sense, it's not really a unit and as-is it looks awkward at best in the UI. `HelpText` with more information would be more valuable.

Reply:
The idea was to separate the status filters that apply only to the users watching status, versus the media airing status.

When they are clumped together with no separation on the UI, it gets confusing (ex: "Import Completed vs "Import Finished')

I was hoping the "Section' would group them visually, but it doesn't seem to do anything at all? I will look into the suggested property. I will remove the unit; I am not even sure what it does, other than add a label to the UI.

Reply:
Help was a better choice. I have updated the labels with prefixes to help differentiate between the two filter sets.